docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/913.md

```diff
@@ -0,0 +1,42 @@
+# Add support for autoscaling filesystem for AWS
+
+Test verifies that for all runs with the file system autoscaling capability being enabled the daemon checks current filesystem usage and attach additional disks in case there is no enough space in the filesystem.
+
+**Prerequisites**:
+- Admin user
+
```

Comment:
Probably, we should add items to the prerequisites - at least, to check that:

- system preference `cluster.instance.hdd.scale.enabled` is enabled
- system preference `cluster.instance.hdd.scale.delta.ratio` is 0.5